project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
@fritsch, @hudokkow: the way it works is that find_package_handle_standard_args takes the version of the library that is found (VERSION_VAR VAAPI_VERSION_STRING) and it will compare that to a bunch of variables that are set by find_package. One of them is VAAPI_FIND_VERSION. So we're already requiring 0.38.0 and the 3 lines you introduced shouldn't be needed.